lib/tasks/dlm.rake

```diff
@@ -0,0 +1,6 @@
+namespace :dlm do
+  desc "Load vendors from Folio"
+  task load_vendor: :environment do
+    VendorLoadService.execute
```

Comment:
Appreciate how you didn't put unique business logic straight in the rake task. 🍻